src/test/java/uk/gov/companieshouse/api/accounts/transformer/CurrentPeriodTransformerTest.java

```diff
@@ -46,22 +59,34 @@ public void testRestToEntityTransformerWithPopulatedObject() {
         currentPeriod.setBalanceSheet(balanceSheet);
 
         CurrentPeriodEntity currentPeriodEntity = currentPeriodTransformer
-                .transform(currentPeriod);
+            .transform(currentPeriod);
+
+        CurrentPeriodDataEntity data = currentPeriodEntity.getData();
 
         Assertions.assertNotNull(currentPeriodEntity);
-        Assertions.assertEquals("etag", currentPeriodEntity.getData().getEtag());
-        Assertions.assertEquals(Integer.valueOf(5),
-                currentPeriodEntity.getData().getBalanceSheetEntity()
-                        .getCalledUpShareCapitalNotPaid());
-        Assertions.assertEquals("kind", currentPeriodEntity.getData().getKind());
-        Assertions.assertEquals(new HashMap<>(), currentPeriodEntity.getData().getLinks());
+        Assertions.assertEquals("etag", data.getEtag());
+        Assertions.assertEquals(Integer.valueOf(CalledUpShareCapitalNotPaidValid),
+            data.getBalanceSheetEntity()
+                .getCalledUpShareCapitalNotPaid());
+        Assertions.assertEquals(Integer.valueOf(TangibleValid),
+            data.getBalanceSheetEntity().getFixedAssets().getTangible());
+        Assertions.assertEquals(Integer.valueOf(FixedAssetsTotalValid),
+            data.getBalanceSheetEntity().getFixedAssets().getTotalFixedAssets());
+        Assertions.assertEquals("kind", data.getKind());
+        Assertions.assertEquals(new HashMap<>(), data.getLinks());
```

Comment:
I would easier to read if the imports were made static to avoid the wrapping. The Integer casting isn't needed.
```java
import static org.junit.jupiter.api.Assertions.assertNotNull;
import static org.junit.jupiter.api.Assertions.assertNull;
import static org.junit.jupiter.api.Assertions.assertEquals;
```
```java
assertNotNull(currentPeriodEntity);
assertEquals("etag", data.getEtag());
assertEquals("kind", data.getKind());
assertEquals(new HashMap<>(), data.getLinks());

BalanceSheetEntity balanceSheetEntity = data.getBalanceSheetEntity();
assertEquals(CALLED_UP_SHARE_CAPITAL_NOT_PAID_VALID, balanceSheetEntity.getCalledUpShareCapitalNotPaid());

FixedAssets fixedAssets = balanceSheetEntity.getFixedAssets()
assertEquals(TANGIBLE_VALID, fixedAssets.getTangible());
assertEquals(FIXED_ASSETS_TOTAL_VALID, fixedAssets.getTotalFixedAssets());
```